client/src/components/Header/components/universal-nav.css

```diff
@@ -77,15 +77,28 @@
   top: 100%;
 }
 
+.exposed-button-nav:is(
+    :hover,
+    :hover:focus,
+    [aria-expanded='true'],
+    [aria-expanded='true']:hover
+  ),
 .lang-button-nav:is(
     :hover,
     :hover:focus,
     [aria-expanded='true'],
     [aria-expanded='false'].force-show
   ) {
-  background-color: var(--gray-10);
+  background-color: var(--gray-00);
   color: var(--theme-color);
-  border: 1px solid var(--gray-00);
+}
+
+.lang-button-nav:is(
+    :hover,
+    :hover:focus,
+    [aria-expanded='true'],
+    [aria-expanded='false'].force-show
+  ) {
   fill: var(--gray-10);
```

Comment:
Btw, do we change the fill to `00`?

```suggestion
  fill: var(--gray-00);
```

![image](https://user-images.githubusercontent.com/88248797/235749831-61c093e7-e103-467f-bb01-0d1cf677f673.png)

Or keep it as is

![image](https://user-images.githubusercontent.com/88248797/235750094-a87bfe4b-3cf5-43c4-8c73-3bbae23d700b.png)

Reply:
I think changing it to -00 is a good Idea.